dev-test/modules/types.ts

```diff
@@ -105,7 +105,13 @@ export type NewStitchingResolver<TResult, TParent, TContext, TArgs> = {
 };
 export type StitchingResolver<TResult, TParent, TContext, TArgs> =
   | LegacyStitchingResolver<TResult, TParent, TContext, TArgs>
-  | NewStitchingResolver<TResult, TParent, TContext, TArgs>;
+  | NewStitchingResolver<TResult, TParent, TContext, TArgs>
+  | StitchingWithoutSelectionSetResolver<TResult, TParent, TContext, TArgs>;
+
+export type StitchingWithoutSelectionSetResolver<TResult, TParent, TContext, TArgs> = {
```

Comment:
Let's call it ResolverWithResolve or something because it is not related to stitching at all.

Reply:
Finally I understand, thanks.
You mean what I really need is a type for an object only has a `resolve` filed, right?

I tried to do that: 85df67c, executed `yarn generate:examples`: 80252ee, and fixed tests: 28f5fea.